tools/autograd/gen_python_functions.py

```diff
@@ -28,7 +28,7 @@
     '_cumsum.*', '_cumprod.*', '_sum.*', '_prod.*', '_th_.*',
     'arange.*', 'range.*', '_gesv.*', '_getri.*', 'slice',
     '_local_scalar', '_local_scalar_dense',
-    'max_pool1d', 'max_pool2d', 'max_pool3d'
+    'max_pool1d', 'max_pool2d', 'max_pool3d', 'linear'
```

Comment:
Why can't we just delete the python linear from functional.py?

Reply:
In general it's better if linear is exposed directly in the same place it was exposed before, but that can be fixed later.